bcgov/HOUS-permit-portal
# Feature/bphh 431/file upload

## Description

Adds step code functionality from the requirement form
(https://hous-bssb.atlassian.net/browse/BPHH-431)

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents
(https://hous-bssb.atlassian.net/browse/BPHH-431)

## Steps to QA
Re-seed
(disable conditionals, this is fixed after we merge to develop)
You should be able to see an energy step code button and clicking it makes the react router transition.
